crowd_anki/representation/note.py

```diff
@@ -28,7 +32,7 @@ def get_notes_from_collection(collection, deck_id, note_models):
     @classmethod
     def from_collection(cls, collection, note_id, note_models):
         anki_note = AnkiNote(collection, id=note_id)
-        note = Note(anki_note)
+        note = Note(anki_note=anki_note)
```

Comment:
👀

Reply:
:thinking: ?

The Note constructor now has two optional parameters:

```python
    def __init__(self, anki_note=None, config: ConfigSettings = None):
        super(Note, self).__init__(anki_note)
        self.note_model_uuid = None
        self.config = config or ConfigSettings.get_instance()
```

So I made sure to be explicit about which optional parameter:
```python
    @classmethod
    def from_collection(cls, collection, note_id, note_models):
        anki_note = AnkiNote(collection, id=note_id)
        note = Note(anki_note=anki_note)
```

Reply:
they are still positional though 🤷‍♂️

Reply:
_Readable and explicit!_ 😁 🤷‍♂